__tests__/index.test.js

```diff
@@ -308,4 +308,18 @@ describe('DELETE /api/comments/:comment_id', () => {
             expect(msg).toBe('Bad Request!')
         })
     });
-})
\ No newline at end of file
+})
+
+describe('GET /api/users', () => {
+    test('GET 200, should return a table of 4 users and each should have 3 keys, username, name and avatar_url', () => {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({body}) =>{
+            expect(body.length).toBe(4)
+            body.forEach((user) =>{
+            expect(Object.keys(user)).toEqual([ 'username', 'name', 'avatar_url' ])
```

Comment:
It would be better to assert the shape of the object and data types. This method uses two iterations rather than one.